inference/core/workflows/execution_engine/v1/executor/runtime_input_assembler.py

```diff
@@ -140,6 +161,65 @@ def _assembly_input_image(
     )
 
 
+def assembly_video_metadata(
```

Comment:
`assembly` is noun, are we preferring verbs for method names? (i.e. `assemble` instead of `assembly`)

Reply:
👍